Lib/test/test_stat.py

```diff
@@ -173,11 +174,16 @@ def test_link(self):
 
     @unittest.skipUnless(hasattr(os, 'mkfifo'), 'os.mkfifo not available')
     def test_fifo(self):
+        if sys.platform == "vxworks":
+            fifo_path = os.path.join("/fifos/", TESTFN)
+        else:
+            fifo_path = TESTFN
         try:
```

Comment:
Please move the self.addCleanup(os_helper.unlink, fifo_path) call before the try. It's ok to call the function on error if the file doesn't exist, the function does nothing in this case.

Reply:
Done.